README.md

```diff
@@ -1,46 +1,65 @@
-# Specialist publisher
-
-## Purpose
+# Specialist Publisher Rebuild
 
 Publishing App for Specialist Documents and Manuals.
 
+## Screenshots
+
+![Homepage/Manuals](docs/screen-shot-specialist-publisher-manuals.png)
+*Homepage/Manuals*
+
+![Specialist Documents - CMA Case Format](docs/screen-shot-specialist-publisher-cma-cases.png)
+*Specialist Documents - CMA Case Format*
+
+![A Manual on gov.uk](docs/screen-shot-manual.png)
+*A Manual on gov.uk*
+
+![A Specialist Document on gov.uk](docs/screen-shot-specialist-document-cma-case.png)
+*A Specialist Document on gov.uk*
```

Comment:
We've been having a few discussions recently about separating out Manuals from Specialist Publisher into a separate app. What do you think about maybe removing these screenshots and adding a note along those lines instead? It might confuse people if we talk about Manuals in the rebuild because I think the plan is changing about what to do with them.